### ISSUE TITLE
8/21 task: Team PSD Weekly Tour & Email Pre-approval Policy for Adjusting Tour

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#1351]
@ritahitching - via email on Wed 4/22/2020

- [x] 1. **Draft a section for the Team PSD Manual that includes the Team PSD weekly tour of duty.**
- The Team PSD weekly schedules are in the Team PSD > teampsd_schedule > "teampsd_weekly_schedule_2020_02_20.docx" 
- They were last updated in February 2020, and could be updated again to reflect our **team_temp** check-ins, which started due to COVID-19, and Rita's adjusted 30/10 tour.

- [x] 2. **Draft a section for the Team PSD Manual that describes the email pre-approval policy, and ADP entry for adjustments to tour.**
- We will secure signatures for this policy during our upcoming 1:1s.

Thanks!

FYI: @anthonycpichardo @staceypark

**Comment by ritahitching** [comment]
@lzim
- [ ]  Draft of [Team PSD Weekly Tour & Email Pre-approval Policy for Adjusting Tour](https://github.com/lzim/teampsd/blob/rita_2020_04_22_weekly_tour_request_change/resources/training_guides/weekly_tour_email_preapproval_policy.md) for discussion and review.
- [ ] Headings:
- Team PSD Tour Work Schedule/Tour of Duty
- Team PSD Standing Meetings  
- Team PSD Core Team Meetings
- Team PSD Process to Change Regular Tour
- Process to Request Vacation Leave
- Process for Sick Leave
- Process to Request Jury Duty Leave
- Process to Request Bereavement Leave
- Process to Request Leave of Absence
- [ ] Updated the team psd weekly schedule to reflect changes since February 
- Added team_temp check-in
- Added changes to Tuesdays (8 hours) and Wednesdays (2 hours) for Dr. Fenn
- Removed weekly training sessions
- Removed weekly team orientation
- New file added to the drive teampsd_weekly_schedule_2020_04_22.docx file

**Comment by lzim** [comment]
@ritahitching 

## Team PSD Weekly Tour (Column A)

- This is a one-page view of all standing meetings, huddles, team time, and wellness breaks following my guidance above. It is already fully captured in Team PSD > teampsd_schedule > "teampsd_weekly_schedule_2020_02_20.docx" with the exception of the edits I mentioned above.
- [ ] Update **team_temp** - I believe this is the only change.
- We don't need individual schedules on GitHub, but it would be great to have your individual schedule updated on Team PSD.

### Team PSD Leave Requests (Column C)
- [ ] Email Lindsey for pre-approval.
- [ ] PAVIR employees put in ADP.
- [ ] Add a link for Team PSD members who are PAVIR employees to refer to the employee handbook

**Comment by ritahitching** [comment]
@ritahitching add a the TeamPSD Logo
- [ ] Leave request
- [ ] Team PSD daily tour
- [ ] Add MTL logo [here](https://github.com/lzim/teampsd/blob/master/resources/logos/team_psd_logo_sm.png)

**Comment by ritahitching** [comment]
@staceypark please review the guide and accept the pull request in issue #1354 when you have time so I can can close this issue and update #1192. 
thanks

**Comment by ritahitching** [comment]
@staceypark 
I updated [Draft Team PSD Weekly Tour & Email Pre-approval Policy for Adjusting Tour](https://github.com/lzim/teampsd/blob/rita_2020_04_22_weekly_tour_request_change/resources/training_guides/weekly_tour_email_preapproval_policy.md) to reflect your feedback on pull request #1354 

- [ ] 1. Rename Pavir file **EE 2017 Handbook.pdf** renamed **2017_employee_handbook_pavir**
- [ ] 2. Convert remaining bullet points converted to **complete sentences** 
- [ ] 3. Add additional **explanation** to ADP codes for Pavir only employees
- [ ] 4. Consolidate section **Process to Change Regular Tour**
- [ ] 5. Wait on clarification of file upload
- [ ] 6. Make new pull request 

**Please Clarify**
To ensure that paths do not break once a pull request is made and approved, please confirm/clarify the best process for uploading documentation that pertains to a branch is to:
1. Create a branch
2. Upload files to the branch
3. Make pull request
4. Wait for files to be merged to master
5. Use master repo GitHub links in any working branch where previously uploaded files were needed for documentation (e.g. file 2017_employee_handbook_pavir.pdf needed for Team PSD Weekly Tour & Email Pre-approval Policy for Adjusting Tour).
6. Only make pull request using master repo GitHub links

**from @staceypark  cross ref pull request #1354** 
1. Rename and reupload the file "EE 2017 Handbook .pdf". This file name does not follow teampsd convention. It is not machine readable because of the mixed cases and spaces. It is not human readable as "EE 2017 Handbook" is not meaningful to anyone.
2. Make sure all your bullets are complete sentences, since that's the format you've followed for most of them.
3. "If approval is granted, changes need to be reflected in ADP with the appropriate code." - This only applies to PAVIR employees as Lindsey outlines in #1351. 
4. Your first three bullet points under "Process to Change Regular Tour" is the same idea rephrased three times. Please combine into one succinct point.
> - [ ] Any scheduled changes to the regular Team PSD tour require written approval in advance. Applies to all types of leave - Vacation, Sick, Jury Duty, Bereavement and Leave of Absence.
> - [ ] No scheduled changes to the tour can be made until approval is granted in writing.
> - [ ] Requests for changes to a regular tour will need to be made formally via email to Dr. Lindsey Zimmerman (i.e. a text message is not sufficient).
5. The images and hyperlinks are all linked within your own branch right now. If you remember from our training, these paths do not update in your pull request and will break. The path always needs to be written to the master branch.

@ritahitching

**Comment by staceypark** [comment]
@ritahitching Thanks for outlining your understanding and proposal of how this would work! 🎉 

**You can do this in one of two ways. I would go with whatever is easiest for you.**
1. You can follow what you outlined, which is to separate the pull request for 1) file upload and 2) file editing

or...

2. You can do the 1) file upload and 2) file editing in one go. Notice the two sections I bolded & italicized in the links below.

- github.com/lzim/teampsd/blob/**_rita_2020_04_22_weekly_tour_request_change_**/resources/training_guides/2017_employee_handbook_pavir.pdf

- github.com/lzim/teampsd/blob/**_master_**/resources/training_guides/2017_employee_handbook_pavir.pdf

- You only need to replace the part of the file or image path that names _**your branch**_ and instead replace it with _**master**_ since that is the **_only file path change that happens in any pull request you make!_** (it's pretty cool right? 😎 )

> **Please Clarify**
> To ensure that paths do not break once a pull request is made and approved, please confirm/clarify the best process for uploading documentation that pertains to a branch is to:
> 
> 1. Create a branch
> 2. Upload files to the branch
> 3. Make pull request
> 4. Wait for files to be merged to master
> 5. Use master repo GitHub links in any working branch where previously uploaded files were needed for documentation (e.g. file 2017_employee_handbook_pavir.pdf needed for Team PSD Weekly Tour & Email Pre-approval Policy for Adjusting Tour).
> 6. Only make pull request using master repo GitHub links

**Comment by ritahitching** [comment]
@staceypark Thank you for the guidance on file naming and merging to master - very helpful.
Please accept the new pull request made #1393.

**Comment by ritahitching** [comment]
@ritahitching Moved issue to May Epic.

**Comment by staceypark** [comment]
@anthonycpichardo merged all these changes last week. This documentation will need updating with our new flex/compress changes and schedule, which we can create a new card for as we get things solidified.

this particular task I believe can be closed.
@lzim @lijenn
